lib/datasource/local/settings_storage.dart

```diff
@@ -130,11 +137,12 @@ class _SettingsStorage {
     await _preferences.setBool(_kIsFirstRun, false);
 
     await _secureStorage.readAll().then((values) {
-      _privateKey = values[_kPrivateKey];
-      _privateKey ??= _migrateFromPrefs(_kPrivateKey);
+      _privateKeysList = values[_kPrivateKeysList]?.split(',') ?? [];
+      _privateKey = privateKeysList?.first;
+      _privateKey ??= _migrateFromPrefs(_kPrivateKeysList); // <-- Migrate what? we don't save private key in pref
```

Comment:
The method to migrate values from shared pref to secure storage sounds cool but none of the 2 fields it is used in are in shared preferences.
In fact, what fields of shared-preferences should it be if required to migrate to secure storage?

Reply:
If we can leave it as it was, it's best, as it concerns keys that were used in previous versions of the app which were stored in settings. Chances are that's so long ago nobody is affected by now - but can't hurt to leave it in

We never, ever, ever want to lose a user's private key with an app update.

Reply:
Ohh I see, It was to migrate past versions, agree I do not see bad leaving those instructions but I will leave them comments about why they are there.